service/controller/internal/cloudconfig/master_template.go

```diff
@@ -17,46 +15,46 @@ import (
 
 // NewMasterTemplate generates a new master cloud config template and returns it
 // as a string.
-func (c *CloudConfig) NewMasterTemplate(ctx context.Context, customObject v1alpha1.AWSConfig, clusterCerts certs.Cluster, clusterKeys randomkeys.Cluster) (string, error) {
```

Comment:
The number of parameters was getting excessive so I created a struct to pass them all in together.